### DISCUSSION TITLE
Provide Import Examples of all supported Data Types

### DISCUSSION THREAD

**Discussion by BenParizek** [engram-design/FeedMe#906] [Ideas]
Currently, FeedMe gives us the option to import data with the following types:

- ATOM
- CSV
- Google Sheet
- JSON
- RSS
- XML

However, in many places in the docs, the only examples given for importing data are provided for `XML` and `JSON`:
- https://docs.craftcms.com/feed-me/v4/guides/importing-assets.html
- https://docs.craftcms.com/feed-me/v4/content-mapping/field-types.html#assets

While some data types may be similar, it would be nice to see all data types addressed in the examples in the docs, even if that is saying they will behave the same as another data type. In particular, it might be nice to see examples of how to structure CSV data for more complex examples like Assets with multiple files or Matrix Fields, or at least some paragraph speaking to whether it is possible. 

Currently, searching "CSV" in the FeedMe documentation returns zero results.

**Comment by outline4** [comment]
Yes, please! 

Like how can you import more than one image when using csv?

**Comment by AMandujano-BI** [comment]
Also stuck with importing more than one image when using Google Sheets.
Would appreciate the help, thanks!

**Comment by outline4** [comment]
I have found the solution in a different thread. 

If you want to import multiple images you have to separate them with the value of 'dataDelimiter' (-|-) which is described here: 

https://docs.craftcms.com/feed-me/v4/get-started/configuration.html#control-panel

so  
image1.png-|-image2.png-|-image3.png  
should do it. 

This is not really documented!  
Please add some examples to the docs!

**Comment by stevenvandemoortele** [comment]
+1 
currently stuck on importing into matrix fields from a google sheet. Docs only have xml / json examples

**Comment by ax2000** [comment]
+1
Documentation around Commerce variant imports using CSV files would be great as well.
